test/Typewriter.Test/TestDataCSharp/com/microsoft/graph/requests/IUnifiedRoleEligibilityRequestRequest.cs

```diff
@@ -0,0 +1,107 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+// **NOTE** This file was generated by a tool and any changes will be overwritten.
+// <auto-generated/>
+
+// Template Source: IEntityRequest.cs.tt
+
+namespace Microsoft.Graph
+{
+    using System;
+    using System.IO;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Linq.Expressions;
+
+    /// <summary>
+    /// The interface IUnifiedRoleEligibilityRequestRequest.
+    /// </summary>
+    public partial interface IUnifiedRoleEligibilityRequestRequest : IBaseRequest
```

Comment:
Are the two 'Request' at the end of IUnifiedRoleEligibilityRequestRequest  expected?

Reply:
yes.